README.md

```diff
@@ -176,7 +200,20 @@ The library is at `build_cmake/libCouchbaseLiteC.so`. (Or `.DLL` or `.dylib`)
 
 ### With CMake on Windows
 
-_(Much like building on Unix. Details TBD)_
+Dependencies:
+* [GCC 7+](http://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/installer/mingw-w64-install.exe/download)
+* [CMake 3.9+](https://github.com/Kitware/CMake/releases/download/v3.19.0-rc1/cmake-3.19.0-rc1-win64-x64.msi)
+* [Git 2.xx.x](https://git-scm.com/download/win)
```

Comment:
Git isn't necessary if you download the source as a zip archive. We didn't list it as a dependency for other platforms. Please remove this line.

Reply:
Technically true, but zip archives I don't think will ever work correctly because they lack submodule support.  However, if you are looking at a GitHub page and don't realize you need to be using git then I doubt you will make it far trying to build / use the library so it's not worth including.